.npmignore

```diff
@@ -10,3 +10,5 @@ schemas
 CODE_OF_CONDUCT.md
 CONTRIBUTING.md
 GOVERNANCE.md
+utils
+scripts
```

Comment:
Borrowed from #9257

Reply:
Perfect! I see test/ is already excluded.